flow-typed/redux/settings.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+// @TODO We are saving the temporary data for Settings page here,
+//  structure will change when modules will added.
+declare type SettingsState = {|
+  +localCurrencyCode: CurrencyCode, // USD
+  +defaultGasPrice: BalanceString,
+  +systemLanguageCode: LanguageCode, // en
+  +hasPinCode: boolean, // false
+|}
```

Comment:
for types these comments are useless.
moreover, it is really easy forget to change them, when default values are updated

Reply:
I mean `// USD` etc